src/main/java/core/db/dailyReport/test33

```diff
@@ -0,0 +1,3 @@
+fruit,quantity
+banana,152
+apple,90
```

Comment:
delete all this files and use just one input and one output files with appropriate path